.changeset/eight-poets-count.md

```diff
@@ -0,0 +1,16 @@
+---
+'@leafygreen-ui/tabs': major
+---
+
+[LG-4087](https://jira.mongodb.org/browse/LG-4087)
+
+- Removed `@leafygreen-ui/portal` dependency and tabs and their corresponding panels are hydrated with initial render
+- Added `@leafygreen-ui/descendants` dependency to handle relationships between `Tabs` and `TabTitle` descendants and between `Tabs` and `TabPanel` descendants
+- Replaced `Box` with `Polymorphic` which allows tab elements to render as any element or component. It is recommended to continue to use a button or link for accessibility purposes
+- Added `forceRenderAllTabPanels` to `Tabs` component. By default, tab panels conditionally render in the DOM. Setting this prop to `true` will render all, non-disabled tab panels in the DOM and hide the non-selected tabs with CSS
```

Comment:
have we verified that these hidden panels don't register to screen readers until they need to be?

Reply:
yes `display:none;` removes it from the accessibility tree. I added a note on this to the changelog 

source: https://developer.mozilla.org/en-US/docs/Web/CSS/display#display_none